iothub/device/tests/Authentication/DeviceClientConnectionStringExceptionTests.cs

```diff
@@ -37,7 +37,7 @@ public void DeviceClientConnectionStringDefaultScopeDefaultCredentialTypeMissing
         }
 
         [TestMethod]
-        [ExpectedException(typeof(ArgumentException))]
+        [ExpectedException(typeof(InvalidOperationException))]
```

Comment:
Is the client documented with this new exception type?

Reply:
Good point, I'll check to make sure.